tests/sample_from_weighted_Shaydulin_Lotshaw_2022.json

```diff
@@ -1327,4 +1327,4 @@
             ]
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
There should be no diff to this file. Could you check out the version from main so that the diff is empty?